src/data/subjects.js

```diff
@@ -1,4 +1,47 @@
 export const commonSubjects = [
+  {
+    longName: {
+      en: 'Engelsk 1',
+      nb: 'Engelsk 1',
+      nn: 'Engelsk 1',
+    },
+    name: {
+      en: 'Engelsk 1',
+      nb: 'Engelsk 1',
+      nn: 'Engelsk 1',
+    },
+    subjectId: 'urn:subject:1:c8d6ed8b-d376-4c7b-b73a-3a1d48c3a357',
+    id: 'programme_subject_c8d6ed8b-d376-4c7b-b73a-3a1d48c3a357',
```

Comment:
Kom på en ting. Om du bruker samme verdi i subjectId og id, så vil programmes.js henvise til faktiske ider fra taksonomi. Då trengs ikkje den fila oppdateres i det heile når vi går over til å hente navn på fag fra taksonomi i stedet for å bruke hardkoda liste.

```suggestion
    subjectId: 'urn:subject:1:c8d6ed8b-d376-4c7b-b73a-3a1d48c3a357',
    id: 'urn:subject:1:c8d6ed8b-d376-4c7b-b73a-3a1d48c3a357',
```

Reply:
Så, eigenteleg bare å søke etter
programme_subject_ 
og erstatte med 
urn:subject:1:
både i subject.js og programmes.js allereie no?